docs/more-info/build-server-setup/tfs-build-vnext.md

```diff
@@ -37,6 +37,13 @@ The VSO build step updates the build number automatically to the GitVersion numb
 GitVersion passes variables in the form of `GitVersion.*` (Eg: `GitVersion.Major`) to TFS Build and also writes `GITVERSION_*` (Eg: `GITVERSION_MAJOR`) environment variables that are available for any subsequent build step. 
 See [Variables](/more-info/variables/) for an overview of available variables.
 
+
+#### Using GitVersion variables in build name
+To use GitVersion's variables in the build name, just add them in the form `$(GITVERSION_FullSemVer)` into the Build definition's build number string. Then just ensure GitVersion is called with
+`/output buildserver` and it will replace those variables with the calculated version.
+The TFS GitVersion Build Step (above) handles this too, so if you're already using that, there's nothing extra to configure.
+
```

Comment:
A note about how the $(rev:r) variable behaves might be useful, since it most probably wont to what the user expects. It increases every time as long as the version number *expanded at build start* is the same. Which is always true for the $(GITVERSION_*) variables.